pkg/permissions/set.go

```diff
@@ -211,3 +211,52 @@ func (ps Set) HasSameRules(other Set) bool {
 
 	return true
 }
+
+// Diff returns a the differences between two sets.
+// Useful to see what rules had been added between a original manifest
+// permissions and now.
+//
+// TODO: We are ignoring removed values/verbs between rule 1 and rule 2.
+// - At the moment, it onlys show the added values & verbs
+// - It does not handle new rules added
```

Comment:
I think we need to handle new rules added: this is how intents work, they add a rule, and it is also how the home adds permissions for on a folder for a konnector